lib/location/scan_wifi.c

```diff
@@ -59,6 +59,42 @@ void scan_wifi_details_get(struct location_data_details *details)
 }
 #endif
 
+static int shutdown_wifi(struct net_if *iface)
+{
+	LOG_DBG("Shutting down Wi-Fi interface");
+	int ret;
+
+	if (!net_if_is_admin_up(iface)) {
+		return 0;
+	}
+
+	ret = net_if_down(iface);
+	if (ret) {
+		LOG_ERR("Cannot bring down wifi iface (%d)", ret);
+		return ret;
+	}
+
+	LOG_DBG("Interface down");
+
+	return 0;
+}
+
+static int startup_wifi(struct net_if *iface)
+{
+	LOG_DBG("Starting up Wi-Fi interface");
+	int ret;
+
+	if (!net_if_is_admin_up(iface)) {
+		ret = net_if_up(iface);
```

Comment:
These should cause Location library test failure, I hope 😄

Reply:
Would you like this to be changed?

Reply:
Nope, just likely needs a fix in the tests and hope it's not too tough as some of the mocking has been painful.

Reply:
Seems location library unit tests fail as expected.
In addition there are some out of memory compilation issues in ATv2 WiFi configurations.
I'll resolve this discussion as it's handled by getting CI to PASS.